src/components/Item/SearchBar.js

```diff
@@ -28,23 +31,39 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-export default function CustomizedInputBase() {
+function CustomizedInputBase(props) {
   const classes = useStyles();
+  const history = useHistory();
+  const [input, setInput] = useState("");
+
+  const handleSubmit = async () => {
+    await props.loadPostingsByQuery(`search=${input}`);
+    history.push({
+      pathname: "/items",
+      search: `search=${input}`,
```

Comment:
could you double check if the search field automatically doesnt add the "search=" prefix to the path as opposed to explicitly being added here. Just wondering

Reply:
Nope, it doesn't add it